### PR TITLE
[author] [new] Added countly-sdk-web@15.08.0

### PR DESCRIPTION
**Pull request by ar2rsawseen**
Origin: https://github.com/Countly/countly-sdk-web

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@ar2rsawseen I'm sorry but we need a high notable popularity to host this lib.

**Comment by PeterDaveHello** [comment]
I'll respectfully suggest you to submit a new issue ticket when it has more than 100 stars on GitHub or has more than 300 downloads per month on npm stats, thanks.

**Comment by ar2rsawseen** [comment]
Ok, thank you for clarification, but it would be good to mention that on contribution description :)

**Comment by PeterDaveHello** [comment]
@ar2rsawseen Oh it's in readme, i will add it in contribution doc, thank you.

**Comment by PeterDaveHello** [comment]
@ar2rsawseen I just found that other related libs of your project/organization have notable popularity, so we can add this lib, just need you to fix the error in CI, thank you :smile:

**Comment by ar2rsawseen** [comment]
Thank you
Rebased and updated initial commit :)

**Comment by ar2rsawseen** [comment]
hmm, npm install failed, seems to be CI configuration problem?

**Comment by PeterDaveHello** [comment]
> hmm, npm install failed, seems to be CI configuration problem? 

Yes, I'll figure it out.

File: ajax/libs/countly-sdk-web/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "keywords": [
+    "Countly",
+    "Analytics",
+    "Web"
+  ],
+  "author": "Countly <hello@count.ly> (https://count.ly/)",
+  "license": "https://github.com/Countly/countly-sdk-web/blob/master/LICENSE",
+  "homepage": "https://count.ly/",
+  "dependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/countly-sdk-web/package.json]: This line can be removed.

File: ajax/libs/countly-sdk-web/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "type": "git",
+    "url": "https://github.com/Countly/countly-sdk-web"
+  },
+  "keywords": [
+    "Countly",
+    "Analytics",
+    "Web"
+  ],
+  "author": "Countly <hello@count.ly> (https://count.ly/)",
+  "license": "https://github.com/Countly/countly-sdk-web/blob/master/LICENSE",
```
Comment (PeterDaveHello) [inline ajax/libs/countly-sdk-web/package.json]: Would you please what kind of license is this? A customized license? Or a well known public license? Thanks.

**Comment by ar2rsawseen** [comment]
Basically license is MIT, no restrictions
removed the line and modified license

**Comment by PeterDaveHello** [comment]
Thanks @ar2rsawseen

**Pull request merged**